drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -371,3 +371,12 @@
     -ms-transform: rotate(45deg);
     transform: rotate(45deg);
 }
+
+.xblock--drag-and-drop--editor .xblock-editor-error-messages {
+    color: darkred;
+}
+
+.xblock--drag-and-drop--editor .xblock-editor-error-messages ul li {
+    list-style-type: disc;
+    margin-left: 20px;
+}
```

Comment:
@haikuginger The `xblock-editor-error-messages` class doesn't seem to be used anywhere; I think you can remove the rules targeting it.